### DISCUSSION TITLE
Install Octopi (Octoprint) with PINN dual boot with LibreELEC

### DISCUSSION THREAD

**Discussion by zillion42** [procount/pinn#689] [General]
Hi,

I recently went on holiday and used my Octopi as a LibreELEC for the Catsitter who was taking care of our cats. I made a backup of everything, so all is fine, but I'm kind of low of Micro SD Cards and I wondered if it were possible to install Octopi (Octoprint) along with a another OS on the same SD Card using PINN.
I know everyone would recommend me using a second SD Card but, that is the whole point, I want avoid that. Most of them are too large anyway, it's kind of waste, and a hassle to change them. I keep physically breaking them when trying to fumble them out of my very inconvenient SD Card reader. Happened just yesterday. Would be very nice to simply boot Kodi before holiday and Octopi after holiday.

I tried to follow this tutorial:

[GitHub](https://github.com/procount/pinn/wiki/How-to-Create-a-Multi-Boot-SD-card-out-of-2-existing-OSes-using-PINN)

[How to Create a Multi Boot SD card out of 2 existing OSes using PINN ·...](https://github.com/procount/pinn/wiki/How-to-Create-a-Multi-Boot-SD-card-out-of-2-existing-OSes-using-PINN)
An enhanced Operating System installer for the Raspberry Pi - How to Create a Multi Boot SD card out of 2 existing OSes using PINN · procount/pinn Wiki

Now I failed because I need alot of Partition information in json files and what not else, I think I read that Octopi is based on Raspbian, could I use those jsons ?

$ cd ~/os/Raspbian  
$ wget -N "http://downloads.raspberrypi.org/raspbian/os.json"  
$ wget -N "http://downloads.raspberrypi.org/raspbian/Raspbian.png"  
$ wget -N "http://downloads.raspberrypi.org/raspbian/partitions.json"  
$ wget -N "http://downloads.raspberrypi.org/raspbian/marketing.tar"  
$ wget -N "http://downloads.raspberrypi.org/raspbian/partition_setup.sh"  
Could someone point me in the right direction please.

Thanks

**Comment by procount** [comment]
PINN would certainly allow you to do this.
If Octopi is based on Raspbian, then you can certainly use those files as a starting point. You will need to take care of some of those parameters in os.json and partitions.json as the tutorial indicates. For a local installation from USB, you will probably need to expand marketing.tar into the slides_vga folder.
os.json is largely descriptive, so it shouldn't be too difficult to work out.
partitions.json indicates how big the partitions need to be at a minimum for your OS. The tutorial indicates how to calculate these values.

If you get stuck, just ask here.
